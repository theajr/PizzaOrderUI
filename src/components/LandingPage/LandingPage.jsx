import React from 'react';
import { Link } from '@reach/router';

const LandingPage = () => {
  return (
    <div style={{ textAlign: 'center' }}>
      <img
        style={{ maxWidth: '100%', height: 'auto' }}
        src="https://images.vexels.com/media/users/3/157292/isolated/preview/1013ce3cf5903ef5afe01f0fd443dc03-colorful-whole-pizza-design-by-vexels.png"
      ></img>
      <h3>This is a pizza ordering application. </h3>
      <h4>
        Please <Link to="/login">login </Link>to proceed!!
      </h4>
    </div>
  );
};

export default LandingPage;
