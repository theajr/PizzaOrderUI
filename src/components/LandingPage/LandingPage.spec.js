import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from './LandingPage';
import { renderComp } from '../../../tests/utilRender';

describe('<LandingPage />', () => {
  test('renders', () => {
    const wrapper = renderComp(<LandingPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
