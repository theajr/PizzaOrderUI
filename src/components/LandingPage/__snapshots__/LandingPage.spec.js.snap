// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPage /> renders 1`] = `ShallowWrapper {}`;
